rubin_sim/maf/metrics/surfbMetric.py

```diff
@@ -0,0 +1,180 @@
+import numpy as np
+from .baseMetric import BaseMetric
+from rubin_sim.photUtils import Bandpass, PhotometricParameters
+import rubin_sim.utils as rsUtils
+from rubin_sim.data import get_data_dir
+import os
+
+__all__ = ["SurfaceBrightLimitMetric"]
+
+
+def surface_brightness_limit_approx(
+    zp, k, airmass, mu_sky, rn=8.8, pixscale=0.2, nsigma=3.0, t_exp=30.0, tot_area=100.0
+):
+    """Compute surface brightness limit in 3 limiting cases, return the brightest.
+
+    Algerbra worked out in this notebook: 
+    https://github.com/yoachim/21_Scratch/blob/main/surfb_metric/surface_brightness_limit.ipynb
```

Comment:
The derivation should probably go in a note, or at least a repo outside of your personal github account.

I have not checked the math, except to notice that the "combining limiting magnitudes" section is a re-derivation of the definition of and standard use-case for t_eff.

Reply:
Good idea, tech note made. I think I broke the sims tech notes by abandoning one a few months ago, so I'll ping the docs folks so it actually gets published.